tests/Functional/EventListener/TagListenerTest.php

```diff
@@ -47,6 +47,23 @@ public function testAnnotationTagsAreSet()
         $this->assertEquals('item-123', $response->headers->get('X-Cache-Tags'));
     }
 
+    /**
+     * @requires PHP 8.0
```

Comment:
oh, i did not know that one. cool!

i was looking for some `$this->markSkipped` call...